site-scrapers/Hannaford.js

```diff
@@ -12,10 +12,9 @@ module.exports = async function GetAvailableAppointments(browser) {
         const response = webData[loc.zip];
         return {
             name: `${siteName} (${loc.city})`,
-            hasAvailability: false, // with current impl of rxTouch, we can't determine for sure if there's availability yet 
-            extraData: response && response.length
-                ? response.substring(1, response.length - 1) //take out extra quotes
-                : response,
+            hasAvailability: Object.keys(response.availability).length != 0 ? true : false,
+            extraData: response.message,
```

Comment:
are we ok with this showing up on the front end?

Reply:
If so, I have some copy suggestions but I'll wait haha

Reply:
yes I imagined this would show on the frontend, which would provide some messaging to the user and would also help us debug in some cases. made some modifications (added `code 1` / `code 2` to help us diagnose any issues that arise)